spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/CompositeEnvironmentRepository.java

```diff
@@ -60,7 +65,12 @@ public Environment findOne(String application, String profile, String label, boo
 		}
 		else {
 			for (EnvironmentRepository repo : environmentRepositories) {
-				env.addAll(repo.findOne(application, profile, label, includeOrigin).getPropertySources());
+				try {
+					env.addAll(repo.findOne(application, profile, label, includeOrigin).getPropertySources());
+				}
+				catch (Exception e) {
+					log.info("Error adding environment for " + repo);
```

Comment:
After some discussion, the team would like this to be opt-in via configuration. The new behavior could surprise users and it's only a single property on config server, not clients.

So 
```
if log errors = true
   log error
else
  rethrow
```

Reply:
What would you suggest as a good name for such a configuration property? `spring.cloud.config.server.fail-on-composite-error` with a default of `true`? It's a little wordy, but all other names I came up with were equally wordy or not clear. And...that's why I'm asking in case someone has a better idea for a name.

Reply:
that name is fine.